src/app/lib/utilities/passport/index.js

```diff
@@ -28,3 +28,19 @@ export const isValidPassportHome = (
 
   return passportHomesOverrideArray.some(matchesPassportHome);
 };
+
+export const getCanonicalUrl = pageData => {
+  const canonicalUrl = pathOr(
+    null,
+    ['metadata', 'locators', 'canonicalUrl'],
+    pageData,
+  );
+  return pageData && canonicalUrl
```

Comment:
is `pageData` needed here?

Reply:
Soz, we copied and pasted it from another function.

Reply:
Good spot! No it's not. I have removed 🙂

It snuck in there as we originally duplicated the getPassortHome function in the same file.

Reply:
noice, thaaaanks